finetuner/tuner/__init__.py

```diff
@@ -52,29 +54,33 @@ def fit(
     device: str = 'cpu',
     callbacks: Optional[List['BaseCallback']] = None,
     num_workers: int = 0,
+    limit: int = 20,
+    distance: str = 'cosine',
```

Comment:
you mean the user building the evaluator and then passing it to fit? We want that? How about `eval_limit` and `eval_distance`?

Reply:
I guess at one point we will also allow the user to specify which metrics to include, and maybe other evaluator kwargs. So then the question is do we copy them all to tuner, or do we abstract them away in the Evaluator instance. What do you think?

Reply:
I think since we decided to integrate the evaluator we should go all the way. Which means expose the eval related args in fit. As a user I don't like having to build the evaluator before calling fit. We also lose the one-line execution of `finetuner.fit`

Reply:
I thought a bit about this - maybe it makes more sense to implement this as a callback. I wrote more in detail in a comment below.